Setup.bat

```diff
@@ -122,6 +122,7 @@ call :MarkStartOfBlock "Retrieve dependencies"
     call :ExecuteAndCheck spatial package retrieve worker_sdk    c_headers                                  %PINNED_CORE_SDK_VERSION%    %DOMAIN_ENVIRONMENT_VAR%    "%CORE_SDK_DIR%\worker_sdk\c_headers.zip"

```

Comment:
as a sidenote, would we be open to having only one space between each argument? I feel like aligning them encourages reading the code column by column which doesn't make sense (you miss details etc)

Reply:
What do you mean? They all follow the same pattern?
```
 spatial package retrieve [PACKAGE DOMAIN] [PACKAGE NAME] %PINNED_CORE_SDK_VERSION% %DOMAIN_ENVIRONMENT_VAR% [PATH]
```

Reply:
Ideally you'd wrap this in a function with the needed [] arguments.

Reply:
for these specific ones a function would be ideal - you're right. There are some other ones where it wouldn't be though

I agree they follow the same pattern, but it feels unreadable to me (and imo encourages bugs as you don't notice mismatches between arguments etc)